ipa/ipa_ec2.py

```diff
@@ -80,23 +80,31 @@ def __init__(self,
                                           running_instance_id,
                                           test_dirs,
                                           test_files)
-        config_file = self.provider_config or EC2_CONFIG_FILE
+        self.account_name = account_name
 
-        if not account_name:
-            raise EC2ProviderException(
-                'Account required for config file: %s' % config_file
+        if not self.account_name:
+            self.logger.warning(
+                'No account provided. To use the EC2 config file an '
+                'account name is required.'
```

Comment:
If it is truly optional no warning should be generate, maybe an inf that states "No account name provided" but a user should not get a warning for not using optional arguments.

Reply:
Okay I like having some sort of message but maybe debug is the best level. Will switch.